.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @isuruf @jakirkham @kalefranz @mbargull @mcg1969 @mingwandroid @msarahan @mwcraig @ocefpaf @patricksnape @pelson @scopatz
\ No newline at end of file
+* @isuruf @jakirkham @kalefranz @mbargull @mcg1969 @mingwandroid @msarahan @mwcraig @ocefpaf @patricksnape @pelson @scopatz @jezdez
```

Comment:
JFYI this is handled through re-rendering

Reply:
Ah, got it, thanks for letting me know!